src/Microsoft.ComponentDetection.Detectors/go/GoComponentDetector.cs

```diff
@@ -247,6 +254,43 @@ private void PopulateDependencyGraph(string goGraphOutput, ISingleFileComponentR
             }
         }
 
+        private bool IsModuleInBuildList(ISingleFileComponentRecorder singleFileComponentRecorder, GoComponent component)
+        {
+            return singleFileComponentRecorder.GetComponent(component.Id) != null;
+        }
+
+        private void RecordBuildDependencies(string goListOutput, ISingleFileComponentRecorder singleFileComponentRecorder)
+        {
+            // Yes, go always returns \n even on Windows
```

Comment:
Is this documented anywhere in the Go docs?

Reply:
Nope (they lack quite a bit of doc tbh, and what's out there can be confusing), me running the commands in windows saw the line endings.